core/operation/src/main/java/uk/gov/gchq/gaffer/named/view/serialisation/NamedViewTypeReference.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016-2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.named.view.serialisation;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import uk.gov.gchq.gaffer.commonutil.iterable.CloseableIterable;
+import uk.gov.gchq.gaffer.data.elementdefinition.view.NamedView;
+
+/**
+ * {@link TypeReference} objects for named operations.
+ */
+public final class NamedViewTypeReference {
```

Comment:
Perhaps just rename this class to TypeReferenceImpl so it can be used for other TypeReferences in the data module.